flags.css

```diff
@@ -2,18 +2,32 @@
 .flagContainer{
    width: 50vmin;
    height: 33vmin;
+   margin: 3vmin;
 }
-#indonesia > .halfRectangle{
+ .halfRectangle{
    height: 50%;
 }
 .red {
-  background-color: red;
+  background-color: #f00;
 }
 .white {
-  background-color: whitesmoke;
+  background-color: #f5f5f5;
 }
-
 /* Colombia */
+.quarterRectangle {
+  height: 25%;
```

Comment:
this is cool but I probably would have reused half rectangle in a parent half rectangle